src/dotnet/commands/dotnet-new/CSharp_Xunittest/$projectName$.csproj

```diff
@@ -12,22 +12,13 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETCore.App">
-      <Version>1.0.1</Version>
-    </PackageReference>
-    <PackageReference Include="Microsoft.NET.Sdk">
-      <Version>1.0.0-alpha-20161104-2</Version>
+    <PackageReference Include="Microsoft.NETCore.App" Version="1.0.1" />
+    <PackageReference Include="Microsoft.NET.Sdk" Version="1.0.0-alpha-20161104-2">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.NET.Test.Sdk">
-      <Version>15.0.0-preview-20161024-02</Version>
-    </PackageReference>
-    <PackageReference Include="xunit">
-      <Version>2.2.0-beta4-build3444</Version>
-    </PackageReference>
-    <PackageReference Include="xunit.runner.visualstudio">
-      <Version>2.2.0-beta4-build1194</Version>
-    </PackageReference>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0-preview-20161109-01" />
```

Comment:
@piotrpMSFT in current `HEAD` eb8e0cfa409d2ef89091f2058487be0e48e8a32e the Microsoft.NET.Test.Sdk is `15.0.0-preview-20161123-03` not `15.0.0-preview-20161109-01` ( ref [xunit template line](https://github.com/dotnet/cli/blob/eb8e0cfa409d2ef89091f2058487be0e48e8a32e/src/dotnet/commands/dotnet-new/CSharp_Xunittest/%24projectName%24.csproj#L23) )